assets/css/styleguide.scss

```diff
@@ -832,7 +852,7 @@ h3 + .button_wrapper {
   padding-bottom: .8rem;
 }
 
-h6.usa-heading-alt {
+h6, h6.usa-heading-alt {
```

Comment:
Prefer to use a class if we need to change layout bc of the styleguide. Unless it's a global rule. Then it should be done in typography.scss.